server/models/Traits.js

```diff
@@ -2,7 +2,39 @@ const { Schema, model } = require('mongoose');
 
 const traitSchema = new Schema(
     {
-
+        name: {
+            type: String,
+            required: true,
+            max_length: 50,
+        },
+        description: {
+            type: String,
+            required: true,
+        },
+        race: {
+            type: String,
+            required: true,
+        },
+        class: {
+            type: String,
+            required: true,
+        },
+        background: {
+            type: String,
+            required: true,
+        },
+        languages: {
```

Comment:
Languages, proficiencies, and feats should be changed to arrays of strings, `type: [String],`  because a character could have multiples of any.

Reply:
Done!